<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="0d7831a7-626b-47da-822d-975df806dba4" name="raffaeu@RAFFAELEGARD400 2011-03-20 22:46:48" runUser="RAFFAELEGARD400\raffaeu" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="364e95ac-57a3-4f1a-b5c1-4676e9ef06b8">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="raffaeu_RAFFAELEGARD400 2011-03-20 22_46_48" />
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2011-03-20T22:46:48.3457031-03:00" queuing="2011-03-20T22:46:48.7988281-03:00" start="2011-03-20T22:46:49.3144531-03:00" finish="2011-03-20T22:46:51.7207031-03:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="AssertThatCanAddANewPerson" storage="c:\users\raffaeu\desktop\book\apress\chapter 14\code\sampleapplication\sampleapplication.fixtures\bin\debug\sampleapplication.fixtures.dll" id="30e11708-3b7a-d6c2-d176-f39bfa7e4374">
      <Execution timeOut="1800000" id="878cf38e-08b9-4678-930f-82b2025b00f0" />
      <TestMethod codeBase="C:/Users/raffaeu/Desktop/Book/APRESS/Chapter 14/Code/SampleApplication/SampleApplication.Fixtures/bin/Debug/SampleApplication.Fixtures.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="SampleApplication.Fixtures.PersonServiceFixture, SampleApplication.Fixtures, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="AssertThatCanAddANewPerson" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="30e11708-3b7a-d6c2-d176-f39bfa7e4374" executionId="878cf38e-08b9-4678-930f-82b2025b00f0" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="878cf38e-08b9-4678-930f-82b2025b00f0" testId="30e11708-3b7a-d6c2-d176-f39bfa7e4374" testName="AssertThatCanAddANewPerson" computerName="RAFFAELEGARD400" duration="00:00:02.0875753" startTime="2011-03-20T22:46:49.3769531-03:00" endTime="2011-03-20T22:46:51.5488281-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="878cf38e-08b9-4678-930f-82b2025b00f0">
      <Output>
        <ErrorInfo>
          <Message>Assert.IsTrue failed. </Message>
          <StackTrace>   at SampleApplication.Fixtures.PersonServiceFixture.AssertThatCanAddANewPerson() in C:\Users\raffaeu\Desktop\Book\APRESS\Chapter 14\Code\SampleApplication\SampleApplication.Fixtures\PersonServiceFixture.cs:line 30
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>